pom.xml

```diff
@@ -55,18 +127,90 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
Lucene is, but WildFly requires Java 8. There's no reason for any code in this project to need Java 7.
(there isn't any code other than tests.. but anyway).